docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,185 @@
+---
+slug: quickstart
+append_help_link: true
+title: Scan your first project with Semgrep
```

Comment:
I'm open to other titles for this page

Reply:
Hola. I've encountered repo vs project cases before.

In Semgrep Cloud Platform, repositories are a superset of all the repos that a user can onboard (from their source code manager), whereas a Project is specifically "a repo that has been added to Semgrep".

"Scan your first repository with Semgrep" maybe?

Reply:
We could also just call it ["Quickstart"](https://docs.snyk.io/getting-started/quickstart)